frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseSlideInBelowAction.java

```diff
@@ -152,7 +152,14 @@ public void onFinished() {
 
   // The UI thread-unsafe calls are actually happening within Runnable lambdas
   // which are going to be executed outside of UI thread.
-  @IgnoreUIThreadUnsafeCalls
+  @IgnoreUIThreadUnsafeCalls({
+      "git4idea.ui.branch.GitBranchCheckoutOperation.<init>" +
```

Comment:
Verbose and conspicuous by design

Reply:
Compare `Await.result(..., Duration...)` in Scala &mdash; same purpose, questionable yet allowed construct, made easily visible on purpose